app/src/main/java/org/merakilearn/datasource/UserRepo.kt

```diff
@@ -84,11 +84,9 @@ class UserRepo(
         }
     }
 
-    suspend fun updateProfile(user: LoginResponse.User, referrer: String? = null): Boolean {
+    suspend fun updateProfile(user: LoginResponse.User): Boolean {
         return try {
-            val response = saralApi.initUserUpdateAsync(
-                UserUpdate(user.name, referrer)
-            )
+            val response = saralApi.updateProfileName(user.id.toInt(),UserUpdateName(user.name,user.profilePicture))
```

Comment:
we should use proper spacing between different parameters, helps increase readability.